app/models/team.rb

```diff
@@ -30,6 +30,7 @@ class Team < ApplicationRecord
   #
   # Right now, all the special namespaces are simply marked as hidden.
   scope :all_non_special, -> { where(hidden: false) }
+  scope :global, -> { where(hidden: true).first }
```

Comment:
You are assuming that they are in order, which may not be true. If you are looking for the global hidden team, then you can just use:

```ruby
find_by(name: "portus_global_team_1")
```

Note: this `1` count is valid because we don't support multiple registries.

Reply:
Yeah. This is not definitive yet. Just wanted something that worked first. I'll keep that in mind.